index.php

```diff
@@ -127,6 +127,50 @@
                 <a class="read-more" href="https://usn.ubuntu.com/releases/ubuntu-18.04-lts/">Security Notices</a>
             </div>
         </section>
+        <section id="workflow" class="grey">
+            <div class="grid">
+                <div class="two-thirds">
+                    <h2>Get work done. Or play.</h2>
```

Comment:
Other sections are title case :)

```suggestion
                    <h2>Get Work Done. Or Play.</h2>
```